client/components/Pipeline/IndividualButton.tsx

```diff
@@ -10,14 +10,21 @@ const StyledButton = styled(Button)`
 `;
 
 interface DropdownProps {
-  loggedInPartnerName: string;
+  loggedInPartnerId: string;
+  reloadKanbanCompanies: (
+    currentTeam: any,
+    currentPartnerId: any
+  ) => Promise<void>;
 }
 
 const IndividualButton: React.FunctionComponent<DropdownProps> = (props) => (
   <STAC>
     {(ac) => (
       <a
-        onClick={() => ac.pipeline.setCurrentPartner(props.loggedInPartnerName)}
+        onClick={() => {
+          ac.pipeline.setCurrentPartner(props.loggedInPartnerId);
```

Comment:
We don't do anything with the currentPartnerId on the pipeline container except check if it's 'ALL'. Maybe makes more sense to change it to currentPartnerFirstName and display that on the partnerDropdown to show which partner is selected

Reply:
Or a boolean isShowingAllPartners, or isSetToSpecificPartner

Reply:
Very down for this @KNWR

Reply:
Did it (described in first comment)